language-plutus-core/language-plutus-core.cabal

```diff
@@ -348,6 +349,39 @@ executable plc
         prettyprinter -any,
         optparse-applicative -any
 
+test-suite language-plutus-core-prop-test
```

Comment:
Let's just merge this into the main test component, I can't really see a good reason not to, unless it's really slow.

Reply:
Might need to write a little wrapper to make these into hunit testcases, but I think those are just a wrapper around `IO` anyway.

Reply:
> Let's just merge this into the main test component, I can't really see a good reason not to, unless it's really slow.

It can be really slow, dependent on the search depth. They're more tests that I'd advice running once in a while, not every time you test.

Reply:
What about just setting the search depth low enough that it's bearable and running them with a high search depth occasionally?

Reply:
The current NEAT tests are *way* faster than the current other tests, so I say merge it and worry about splitting it up again only if it's actually a problem.

Reply:
I merged `language-plutus-core-prop-test` into `language-plutus-core-test` and removed `prop-test/Spec.hs` putting the remaining contents into `test/Spec.hs`. I am not sure whether to move the contents of `prop-test` into `test` and if so, where to put it.

Reply:
For now just move all the source files in in the same locations, i.e. `test/Language/PlutusCore/Gen/Common.hs`